pca-main.template

```diff
@@ -350,30 +350,30 @@ Parameters:
       steps required in Amazon QuickSight to (1) enable S3 access to PCA OutputBucket and (2) share dashboard and analytics assets.
 
   GenAIQuery:
-    Default: 'DISABLED'
+    Default: 'BEDROCK'
     Type: String
     AllowedValues:
       - 'DISABLED'
       - 'BEDROCK'
       - 'ANTHROPIC'
     Description: >
       If enabled, gives the ability to query an individual call for information. 
-      The BEDROCK option requires your account to have Amazon Bedrock preview access.
+      The BEDROCK option requires you to choose one of the supported model IDs from the provided list (GenAIQueryBedrockModelId).
       The ANTHROPIC option is a third party service, and you must enter your Anthropic API key in the 
       Third Party LLM API Key section.
 
   GenAIQueryBedrockModelId:
     Type: String
     Default: anthropic.claude-v2
```

Comment:
Why not claude-v1-instant? Seems unintuitive to use v2 as the default for this, but instant as the default for summarization..

Reply:
Also add stack output (as in LCA) with hyperlink to the SSM parameter that contains the prompt, for easy editing..  Update any affected READMEs to explain this..